integration_tests/src/main/python/orc_write_test.py

```diff
@@ -15,11 +15,11 @@
 import pytest
 
 from asserts import assert_gpu_and_cpu_writes_are_equal_collect, assert_gpu_fallback_write
-from spark_session import is_before_spark_320, is_spark_cdh, is_spark_321cdh
+from spark_session import is_before_spark_320, is_spark_cdh, is_spark_321cdh, with_gpu_session, with_cpu_session
```

Comment:
Nitpick: Should the imports be alphabetically sorted in the same line?

Reply:
Done